backend/functions/cases/cases.integration.test.ts

```diff
@@ -0,0 +1,31 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+describe('Integration Test for the cases Azure Function to call Chapter15 cases',()=>{
+  let functionUrl;
+  beforeAll(() => {
+    functionUrl = process.env.CASES_FUNCTION_URL;
+  });
+
+  beforeEach(()=> {
+    jest.setTimeout(300000);
+  });
+
+  test('cases azure function should return success when called with caseChapter 15 and a professionalId',async()=>{
+    jest.setTimeout(300000);
+    const _caseChapter= '15';
```

Comment:
Let's look into these timeouts to see if they are working correctly.  The jest default is to fail if a test takes longer than 5s I believe.  If these timeouts aren't working there may be something else we have to do to override the default of 5000ms.

Reply:
Note the inclusion of `, 15000` on line 24. I validated this works with the use of a sleep function. It will not timeout until we've reached 15 seconds now.